src/main/java/com/google/devtools/build/lib/runtime/commands/CleanCommand.java

```diff
@@ -79,6 +79,17 @@
              + "this command completes, it will be safe to execute new commands in the same "
              + "client, even though the deletion may continue in the background.")
     public boolean expunge_async;
+
+    @Option(name = "async",
+        defaultValue = "false",
+        category = "clean",
+        expansion = "--clean_style=async",
+        help = "If specified, clean will asynchronously remove the entire working tree for "
```

Comment:
I would use present tense here:
"If specified, %product% asynchronously removes the entire working tree ..."

I realize that the other doc doesn't do that right now, and I think those should be changed as well.